### ISSUE TITLE
 Error in stopifnot(is_n(n)) : could not find function "is_n" 

### ISSUE THREAD

**Issue by marcone84** [mkearney/rtweet#124]
Hello, 

I'm trying to use the new "get_retweeters" functions and keep running into this error. Thank you for any help. 

Marcus

**Comment by mkearney** [comment]
Hrmm. There's actually an API limitation that prevents n from being meaningful (results are limited to the first 100), but this error message should basically never come up.

``` r
> x <- get_retweeters("918195755432775680")
> x
              user_id
1          1977619399
2  818366244667748352
3          2565809797
4  714286505305317376
5            15244528
6          2870432036
...
> x <- get_retweeters("918195755432775680", n = TRUE)
Error: is_n(n) is not TRUE
```

Can you share the output from the following code?

``` r
sessionInfo()
packageVersion("rtweet")
rtweet:::is_n
```

Along with the code you used to get that error?

**Comment by marcone84** [comment]
Hi Michael, 

Thank you so much for replying. Turns out I had stupidly altered the function myself and it was responding to this. I fixed it and the code you provided worked. Apologies from a novice. On another note, out of curiosity, have you had any success using the cursor parameter with this command to get all retweeters for a given tweet?

Thank you again for your time. The package is fantastic to work with. 

Best,

Marcus Mann
Ph.D. Candidate, Sociology, Duke University
marcusmann.net <http://marcusmann.net/>
